src/mlpack/tests/ann_visitor_test.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file ann_visitor_test.cpp
+ *
+ * Tests for testing visitors in ANN's of mlpack.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+#include <mlpack/methods/ann/layer/layer.hpp>
+#include <mlpack/methods/ann/layer/layer_types.hpp>
+#include <mlpack/methods/ann/visitor/bias_set_visitor.hpp>
+#include <mlpack/methods/ann/visitor/weight_set_visitor.hpp>
+#include <mlpack/methods/ann/visitor/reset_visitor.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace mlpack;
+using namespace mlpack::ann;
+
+BOOST_AUTO_TEST_SUITE(ANNVisitorTest);
+
+BOOST_AUTO_TEST_CASE(BiasSetVisitorTest)
```

Comment:
If you can add a super simple description what this test does, I think this is ready.

Reply:
Hi @zoq. Done.